ibis/pyspark/compiler.py

```diff
@@ -1162,12 +1162,13 @@ def compile_dense_rank(t, expr, scope, timecontext, **kwargs):
 
 
 @compiles(ops.PercentRank)
-def compile_percent_rank(t, expr, scope, timecontext, **kwargs):
```

Comment:
Not sure about this. But in the past, we are normalizing indexes (-1). Is this approach deprecated?